.gitignore

```diff
@@ -8,3 +8,5 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+.coverage
```

Comment:
Consider removing the space before the `.coverage` entry to maintain consistency with the other entries.